src/examples/elementary/meson.build

```
@@ -123,6 +119,16 @@ examples = [
   'efl_canvas_textblock_obstacles_example'
 ]
 
+# Disabling examples that are directly using pthreads 
+if !sys_windows
+  examples += [
+    'efl_thread_1',
+    'efl_thread_2',
+    'efl_thread_3',
+    'efl_thread_4',
+  ]
```

probably a good idea to use `files`:
```suggestion
  examples += files([
    'efl_thread_1',
    'efl_thread_2',
    'efl_thread_3',
    'efl_thread_4',
  ])
```

---

They are not files.

---

ahhh, right right